### ISSUE TITLE
Automatically hide touch controls when a joystick is detected 

### ISSUE THREAD

**Issue by luisminze** [hrydgard/ppsspp#16752]
### What should happen

Hello !, a function  function that I find strange that PPSSPP has not had in a while, basically it is to automatically hide the touch controls when a Joystick is detected, and when no Joystick is connected ,Touch controls come back normally.

### Who would this benefit

All.

### Platform (if relevant)

Android

### Games this would be useful in

All.

### Other emulators or software with a similar feature

DuckStation
AetherSX2
PizzaBoy GBA 
PizzaBoy

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by NABN00B** [comment]
I'd prefer if this were an option and not being forced, because currently touch controls is the only way to show inputs on screen. This is useful when recording explanation/tutorial/showcase videos.

**Comment by hrydgard** [comment]
You can manually turn them off already, and they fade out after a few seconds if you don't touch the screen, which you presumably won't when playing with a controller - so I'm not sure this is actually all that useful?

**Comment by luisminze** [comment]
Yes, but this way is much more practical, which basically completely hides the touch controls while the joystick is connected, regardless of not touching the screen or not, it's something simple but would be very welcome


Here's an example using AetherSX2, the virtual controller disappears completely while the joystick is connected, even after leaving and entering a game. 

https://user-images.githubusercontent.com/77176224/212439031-4fecb1a4-493a-40db-9d8c-97acc8e67e5c.mp4

**Comment by hrydgard** [comment]
You can change the "Auto-hide buttons after seconds" to a very small number, to get something quite similar effectively.

I know it's not exactly what you want, but it'll allow you to play without waiting too long for them to disappear. Good stop-gap solution.

**Comment by luisminze** [comment]
> You can change the "Auto-hide buttons after seconds" to a very small number, to get something quite similar effectively.
> 
> I know it's not exactly what you want, but it'll allow you to play without waiting too long for them to disappear. Good stop-gap solution.

No problem, it was just to let the team know about this suggestion, If possible of course, to be added in a future version, it will be very welcome.

**Comment by unknownbrackets** [comment]
Personally I think auto-hide practically solves this, and you could just turn off touch controls if you don't want them.  That said, I can see how if you're often touching your screen maybe that wouldn't be ideal.  Seems like an edge case, though...

If we did do this, would probably have to think about wireless controllers temp disconnecting, which is probably a more common edge case.  Sometimes bluetooth devices disconnect for short periods, which is super annoying, and if that made touch controls flash on the screen it'd probably only make it worse...

-[Unknown]

**Comment by hrydgard** [comment]
Yeah, all that plus we need yet another translation string, and for very little benefit above timed auto-hide. I will leave this open for now as it's legimitate as a feature request, but it is not likely to be prioritized.
